examples/redux/src/App.js

```diff
@@ -70,15 +70,29 @@ const Home = () => (
     </header>
     <Link to="/protected1">protected1</Link>
     <span> - </span>
-    <Link to="/protected2">protected2</Link>
+    <Link to="/protected2">
+      protected2{' '}
+      <small>
+        <i>(with custom Authenticating Component)</i>
+      </small>
+    </Link>
     <p>Demo access_token is valid during 30 seconds</p>
     <Button403Enhance />
     <Button401Enhance />
   </div>
 );
 
+const CustomAuthenticatingComponent = () => (
+  <div className="App">
+    <header className="App-header">
+      <h1 className="App-title">Authenticating</h1>
+      <i>rendered with a custom Component</i>
+    </header>
+  </div>
+);
+
 const ProtectedChild = () => (
-  <OidcSecure isEnabled={configuration.isEnabled}>
+  <OidcSecure isEnabled={configuration.isEnabled} authenticatingComponent={CustomAuthenticatingComponent}>

```

Comment:
In the oidc context version i thnink the props name is authenticating, it would be great to set the samedi name.

Reply:
and dont forget to add rhe props info in the readme :)

Reply:
Thanks, i just added a section in Readme.

About `authenticating`, i agree that how it's called in Context package, however it can misinterpret the usage because authenticating sound like a boolean, should we change from now on to `authenticatingComponent` like other props (sessionLostComponent) ? 
Well may be for now i can change to `authenticating` to unify use cases 👍